espresso-server/app/src/androidTest/java/io/appium/espressoserver/lib/helpers/ThreadHelpers.kt

```diff
@@ -0,0 +1,18 @@
+package io.appium.espressoserver.lib.helpers
+
+object ThreadHelpers {
```

Comment:
kotlin allows to create public functions, there is no need for a wrapping object